modules/build/src/test/scala/scala/build/tests/BuildTests.scala

```diff
@@ -678,6 +678,20 @@ class BuildTests extends munit.FunSuite {
 
   }
 
+  test("ScalaNative install managed clang") {
+    val inputs = TestInputs(
+      os.rel / "p.sc" ->
+        """//> using `native-version` "0.4.0"
+          |//> using nativeClangManaged
+          |""".stripMargin
+    )
+    val buildOptions =
+      defaultOptions.copy(internal = defaultOptions.internal.copy(keepDiagnostics = true))
+    inputs.withBuild(buildOptions, buildThreads, bloopConfig) { (_, _, maybeBuild) =>
+      assert(maybeBuild.toOption.get.options.scalaNativeOptions.clangManaged.get == true)
```

Comment:
```suggestion
      assert(maybeBuild.toOption.get.options.scalaNativeOptions.clangManaged.contains(true))
```
(shoudn't throw before the assertion is checked, if ever `clangManaged` is empty)